src/index.css

```diff
@@ -1,27 +1,60 @@
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  margin: 5rem;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.input-filed {
+  background-color: gray;
+  padding: 0.7rem;
+}
+
+.btn-last {
+  width: 50%;
+  padding: 20px;
```

Comment:
- [ ] The styling does not fully match the required one. Please, consider fixing it. 

> Here is your version:
<img width="562" alt="Screen Shot 2022-08-17 at 18 43 50" src="https://user-images.githubusercontent.com/49064106/185168433-a47ea6f9-131b-45e2-9f78-52c6643847b6.png">

> Required one:
<img width="330" alt="Screen Shot 2022-08-17 at 18 43 33" src="https://user-images.githubusercontent.com/49064106/185168262-5f19fe8f-8232-4ba9-9ded-826d8e8dcc98.png">